src/sagas/user.saga.ts

```diff
@@ -45,8 +67,13 @@ function* onboard(action) {
 function* getUserData() {
   try {
     //take the token from localstorage and send in getUser
-    const user = yield userService.getUser();
+    const storedData = JSON.parse(localStorage.getItem("userData"));
 
+    let user = false;
```

Comment:
How can the user be both boolean and object? it opens room for bugs, would suggest change that or just define user so it will be undefined this way if(user) will still return false